src/istgt_lu.c

```diff
@@ -1898,14 +1898,19 @@ istgt_lu_add_unit(ISTGT_Ptr istgt, CF_SECTION *sp)
 				ISTGT_TRACELOG(ISTGT_TRACE_DEBUG, "Device file=%s\n",
 							   lu->lun[i].u.device.file);
 			} else if (strcasecmp(val, "Storage") == 0) {
+				int idx2 = 1;
 				if (lu->lun[i].type != ISTGT_LU_LUN_TYPE_NONE) {
 					ISTGT_ERRLOG("LU%d: duplicate LUN%d\n", lu->num, i);
 					goto error_return;
 				}
 				lu->lun[i].type = ISTGT_LU_LUN_TYPE_STORAGE;
 
-				size = istgt_get_nmval(sp, buf, j, 1);
-				rsz  = istgt_get_nmval(sp, buf, j, 2);
+#ifndef	REPLICATION
```

Comment:
I could not understand how it was working before replication. We were not using file in earlier code but now we are using it. Just want to know little more about "file" concept.